test/unity/toolkits/object_detection/test_object_detector.cxx

```diff
@@ -397,7 +396,7 @@ BOOST_AUTO_TEST_CASE(test_object_detector_train) {
 
   // The following callbacks capture by reference so that they can transfer
   // ownership of the mocks created above.
-
+  /*
```

Comment:
Should this code just be deleted rather than commenting it out? If it should just be commented out, please add a comment about why.

Reply:
This is a work-in-progress PR. Not ready for code clean-up review yet. I wanted Nick's feedback on the logic.

Reply:
When code is ready for a full review, I'll remove the 'WIP' tag.

Reply:
Sounds good.